### PR TITLE
Add wall time data in log

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: sources/Inputs/src/Configuration.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ Configuration::Configuration(const std::string& filepath) {
     helper::updateValue(useLCCAsLoads_, config, "LCCAsLoads");
     helper::updateValue(outputDir_, config, "OutputDir");
   } catch (std::exception& e) {
-    LOG(error) << e.what() << LOG_ENDL;
+    LOG(error) << "Error while reading configuration file: " << e.what() << LOG_ENDL;
```
Comment (lecourtoisflo) [inline sources/Inputs/src/Configuration.cpp:72]: Maybe add the filepath in error log

File: sources/main.cpp [hunk 2]
```diff
@@ -96,31 +103,45 @@ main(int argc, char* argv[]) {
     dfl::Context context(def, config);
 
     if (!context.process()) {
+      LOG(info) << MESS(InitEnd, timerInit.elapsed()) << LOG_ENDL;
```
Comment (lecourtoisflo) [inline sources/main.cpp:106]: we leave with a failure so we should have an error log as last log

File: sources/main.cpp [hunk 3]
```diff
@@ -96,31 +103,45 @@ main(int argc, char* argv[]) {
...
 
-    LOG(info) << MESS(SimulationEnded, context.basename()) << LOG_ENDL;
+    LOG(info) << " ============================================================ " << LOG_ENDL;
+    LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
 
     return EXIT_SUCCESS;
   } catch (DYN::Error& e) {
+    std::cerr << "Simulation failed" << std::endl;
     std::cerr << "Dynawo: " << e.what() << std::endl;
+    LOG(error) << "Simulation failed" << LOG_ENDL;
```
Comment (lecourtoisflo) [inline sources/main.cpp:126]: It is an error log  so we should use the dictionnaries

File: sources/main.cpp [hunk 4]
```diff
@@ -18,7 +18,9 @@
 #include <DYNError.h>
 #include <DYNIoDico.h>
 #include <boost/filesystem.hpp>
+#include <boost/timer.hpp>
 #include <cstdlib>
+#include <iomanip>
```
Comment (lecourtoisflo) [inline sources/main.cpp:23]: I don't see the use of this header

File: sources/main.cpp [hunk 5]
```diff
@@ -96,31 +103,45 @@ main(int argc, char* argv[]) {
...
-
     dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
+    LOG(info) << MESS(FilesEnd, timerFiles.elapsed()) << LOG_ENDL;
+
+    boost::timer timerSimu;
+    context.execute();
 
-    LOG(info) << MESS(SimulationEnded, context.basename()) << LOG_ENDL;
+    LOG(info) << " ============================================================ " << LOG_ENDL;
+    LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
```
Comment (lecourtoisflo) [inline sources/main.cpp:120]: Maybe the last that should appear is the total time, not the simulation time. I was confused the fist time I read the logs

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

**Review by gautierbureau — APPROVED** [review]

**Review by marcochiaramello — APPROVED** [review]

File: sources/Inputs/src/Configuration.cpp (hunk 1 above)
Comment (rosiereflo) [reply sources/Inputs/src/Configuration.cpp:72]: The filepath is given in the exception (at least for the errors that I saw)

File: sources/main.cpp (hunk 3 above)
Comment (rosiereflo) [reply sources/main.cpp:126]: Are we sure that the dictionary is available at this point?

**Review by lecourtoisflo — APPROVED** [review]

**Pull request merged**